index.bs

```diff
@@ -12346,12 +12364,60 @@ It is the responsibility of specifications using Web IDL to state
 which global environment (or, by proxy, which global object) each platform
 object is associated with.
 
+<div algorithm>
+  To create a <dfn export>new</dfn> object implementing the interface |interface|, with a Realm
```

Comment:
I tend to prefer having a longer name for the &lt;dfn&gt; part, like "create a new object implementing an interface", which seems to be what we do with "create an operation function" etc. We can then have a shorter alternative link-text like just "new" or something, with a clarification in a note.

Reply:
Please note the previous discussion about *not* providing multiple names for this.

Reply:
I think what @TimothyGu is suggesting is that we still provide a single convention for callers `Let <var>x</var> be a <a for=/>new</a> <code>Y</code>`, but that the `<dfn>` part here is a bit longer. I'd be okay with that, though some additional rationale would help.